<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>KisDitherWidget</class>
 <widget class="QWidget" name="KisDitherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>154</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item column="0" row="1">
    <widget class="QLabel" name="thresholdModeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Threshold Mode</string>
     </property>
     <property name="buddy">
      <cstring>thresholdModeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item column="1" row="1">
    <widget class="QComboBox" name="thresholdModeComboBox">
     <item>
      <property name="text">
       <string>Pattern</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Noise</string>
      </property>
     </item>
    </widget>
   </item>
   <item colspan="2" column="0" row="3">
    <widget class="KisDoubleSliderSpinBox" name="spreadSpinBox"/>
   </item>
   <item colspan="2" column="0" row="2">
    <widget class="QStackedWidget" name="thresholdModeStackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="prefix" stdset="0">
      <string>Amount:  </string>
     </property>
     <widget class="QWidget" name="patternWidget">
      <layout class="QFormLayout" name="formLayout_2">
       <item column="0" row="0">
        <widget class="QLabel" name="patternLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Pattern</string>
         </property>
         <property name="buddy">
          <cstring>patternIconWidget</cstring>
         </property>
        </widget>
       </item>
       <item column="1" row="0">
        <widget class="KisIconWidget" name="patternIconWidget">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="0" row="1">
        <widget class="QLabel" name="patternValueModeLabel">
         <property name="text">
          <string>Value Mode</string>
         </property>
         <property name="buddy">
          <cstring>patternValueModeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item column="1" row="1">
        <widget class="QComboBox" name="patternValueModeComboBox">
         <item>
          <property name="text">
           <string>Auto</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="Name of Dither pattern value mode - Lightness">Lightness</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="Name of Dither pattern value mode - Alpha">Alpha</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="noiseWidget">
      <layout class="QFormLayout" name="formLayout_5">
       <item column="0" row="0">
        <widget class="QLabel" name="noiseSeedLabel">
         <property name="text">
          <string>Seed</string>
         </property>
         <property name="buddy">
          <cstring>noiseSeedLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item column="1" row="0">
        <widget class="QLineEdit" name="noiseSeedLineEdit"/>
       </item>
       <item column="1" row="1">
        <widget class="QToolButton" name="noiseSeedRandomizeButton">
         <property name="text">
          <string>Randomize</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisIconWidget</class>
   <extends>QToolButton</extends>
   <header location="global">kis_iconwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">kis_slider_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>thresholdModeComboBox</tabstop>
  <tabstop>patternIconWidget</tabstop>
  <tabstop>patternValueModeComboBox</tabstop>
  <tabstop>noiseSeedLineEdit</tabstop>
  <tabstop>noiseSeedRandomizeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>thresholdModeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>thresholdModeStackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thresholdModeStackedWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>thresholdModeComboBox</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>